lib/BackendClient.js

```diff
@@ -154,32 +153,49 @@ var BackendClient = (function(){
         return deferred.promise;
     }
 
-    function getEntryInfo(entryId, serverNodeId) {
+    function getServerType(serverNodeId, resultReceived) {
+        return (resultReceived.primaryServerNodeId === serverNodeId) ? kalturaTypes.KalturaEntryServerNodeType.LIVE_PRIMARY :
+            kalturaTypes.KalturaEntryServerNodeType.LIVE_BACKUP;
+    }
+
+    function parseServerEntryInfo(serverNodeId, apiCallResults) {
+        var liveEntryObject = convertKalturaEntry(apiCallResults[0]);
+        liveEntryObject.serverType = getServerType(serverNodeId, apiCallResults[0]);
```

Comment:
@yossipapi  shouldn't we take the serverType from serverEntryNode instead of liveEntry?